com.civicrm.hrjobroles/js/test/filters/option-group-filter_test.js

```diff
@@ -0,0 +1,59 @@
+define([
+    'common/angular',
+    'common/angularMocks',
+    'job-roles/filters/option-group-filter',
+    'job-roles/filters/filters',
+], function(angular, moment) {
+  'use strict';
+
+  describe('Using getActiveGroup filter', function() {
+    var $filter, locations, getActiveGroup;
+
+    beforeEach(module('hrjobroles.filters'));
+    beforeEach(inject(['$filter', function(_$filter_) {
+        $filter = _$filter_;
+        getActiveGroup = $filter('getActiveGroup');
+      }
+    ]));
+
+    it('should define getActiveGroup filter ', function() {
+      expect($filter('getActiveGroup')).toBeDefined();
+    });
+
+    describe('When filtering locations, departments, region and lavels ', function() {
+      beforeEach(function() {
+        locations = {
+          "Headquarters": {
+            "id": "Headquarters",
+            "title": "Headquarters",
+            "is_active": "1"
+          },
+          "Home": {
+            "id": "Home",
+            "title": "Home or Home-Office",
+            "is_active": "1"
+          },
+          "Other": {
+            "id": "Home",
+            "title": "Home or Home-Office",
+            "is_active": "0"
+          }
+        }
+      });
+
+      it('should have 2 locations after filtration ', function() {
+        expect(getActiveGroup(locations, '1').length).toBe(2);
+      });
+
+      it('should have all data with is_active set to 1 ', function() {
+        expect(getActiveGroup(locations, '1')[0].is_active).toBe('1');
```

Comment:
To be honest I don't like this filter signature.

First of all the name. You're not picking the active "group", you're picking the active "value", aren't you? If that's the case, why isn't the filter then called `getActiveValues`?

Second, if the filter is explicitly about picking the active values, what's the need for the second parameter? You are always passing `1` anyway! And given the name of the filter (get**Active**Group), one would expect that the filter automatically picks the active values by default, without having to pass a parameter to specify that. I don't think the second parameter is required at all